tests/helpers.py

```diff
@@ -207,3 +207,20 @@ def ip_to_device_id_map(clients):
             ip_to_device_id[identity_to_ip[device['device_identity']]] = device['device_id']
 
         return ip_to_device_id
+
+    @staticmethod
+    def get_mender_logs():
+        output_from_deployment_logs = ""
+
+        try:
+            output_from_journalctl = run("journalctl -u mender -l")
+            logs_exist = int(run("find '/data/mender' -name '*.log' -type f | wc -l").strip())
+
+            if logs_exist:
+                output_from_deployment_logs = run("grep '' -H /data/mender/*.log")
+
+        except (BaseException):
+            logger.warn("failed to grab logs from mender client")
+            return None, None
+        else:
+            return output_from_journalctl, output_from_deployment_logs
```

Comment:
`output_from_journalctl` is defined inside the `try`. Do you need one outside to make sure it stays in scope? Like with `output_from_deployment_logs`.

Reply:
That's not how scope works in Python

Reply:
That's not how scope works in Python